ReactTask/src/views/PatientProfile.js

```diff
@@ -0,0 +1,254 @@
+import React, { useEffect, useState } from 'react';
+import {
+  StylBoxRowPadding,
+  StylBoxPatients,
+  StylBoxDoctor,
+  StylBoxFlexColumn,
+  StylBoxColumnFlex,
+  StylBoxPatientContent,
+  StylBoxBtnPatients,
+  StylBoxRow,
+  BtnPatientsHiddenMob,
+  BtnCreateAppointment,
+  BtnPatients,
+  StylSubTitlePosition,
+  StylTitlePatients,
+  StylTitleName,
+  AvatarActive,
+  CardProfilePatient,
+  BoxFilter,
+  StylTitlePatientAppointment,
+  StylTitleCurrentPatientAppointment,
+  StylBoxRowFlexStart,
+  StylChooseDateAndTimeVisitDoctor,
+  StylItemChooseDoctor,
+  StylTitleVisitDoctor,
+  StylInputVisitDoctor,
+  StylArrowVisitDoctor,
+  StylBoxMenuSelectVisitDoctor,
+  StylElementSelectVisitDoctor,
+  StylBoxColumnFlexMargin,
+  CalendarVisitDoctor,
+  StylBoxChooseTime,
+  StylBtnChooseTimeVisitDoctor,
+  StylBtnSubmitFormVisitDoctor,
+} from '../components/';
+
+import patient from '../img/avatars/avatar-patient.png';
+import patient1 from '../img/avatars/patients-13.png';
+import logo from '../img/logo.png';
+import data from './../db/dbProfilePatient.json';
+import {
+  nameDoctorsArr,
+  ocuppationDoctorArr,
+  possibleTimeArr,
+} from './../constants/index';
+
+const PatientProfile = ({ title }) => {
+  const dbData = [data],
+    imgArr = [patient1, patient1, patient1],
+    [isHiddenAppointment, setHiddenAppointment] = useState(false),
+    [isFocusInput, setFocusInput] = useState(false),
+    [isFocusInputName, setFocusInputName] = useState(false),
+    [isValueInputOccupation, setValueInputOccupation] = useState(''),
+    [isValueInputName, setValueInputName] = useState(''),
+    [isFilledData, setFilledData] = useState(false),
+    [isTimeSelect, setTimeSelect] = useState(''),
+    [isChooseAllData, setChooseAllData] = useState(false);
+
+  let isDataChoose = '';
+
+  useEffect(() => {
+    document.title = title || '';
+  }, [title]);
+
+  useEffect(() => {
+    isValueInputName && isValueInputOccupation
+      ? setFilledData(true)
+      : setFilledData(false);
+  }, [isValueInputOccupation, isValueInputName]);
+
+  useEffect(() => {
+    isFilledData && isTimeSelect && isDataChoose
+      ? setChooseAllData(true)
+      : setChooseAllData(false);
+  }, [isFilledData, isTimeSelect, isDataChoose]);
+
+  const handleContent = () => {
+    setHiddenAppointment(!isHiddenAppointment);
+  };
+
+  const changeDataChoose = (data) => {
+    isDataChoose = data;
+  };
+
+  const chooseOccupationDoctor = (e) => {
+    const selectData = e.currentTarget.innerHTML;
+
+    setTimeout(() => {
+      setValueInputOccupation(selectData);
+      setFocusInput(false);
+    }, 200);
+  };
+
+  const chooseNameDoctor = (e) => {
+    const selectData = e.currentTarget.innerHTML;
+
+    setTimeout(() => {
+      setValueInputName(selectData);
+      setFocusInputName(false);
+    }, 200);
+  };
+
+  return (
+    <StylBoxPatients>
+      <StylBoxRow>
+        <img src={logo} alt='logo' />
```

Comment:
If it is header. Rodion said to me that this header we can reuse. Data we need give via props. I'm about for doctor and patient the same header

Reply:
Create component for parts that have been used across the app.

Reply:
Created component `Header profile`

Reply:
Resolved